orocos_kdl/tests/velocityprofiletest.cpp

```diff
@@ -334,3 +334,45 @@ void VelocityProfileTest::TestTrapHalf_SetDuration_End()
 	CPPUNIT_ASSERT_DOUBLES_EQUAL(0.0, v.Acc(time),epsilon);
 
 }
+
+void VelocityProfileTest::TestDirac_SetProfile()
+{
+    double time;
+    double pos1 = 10.0;
+    double pos2 = -1.0;
+
+    VelocityProfile_Dirac v;
+    v.SetProfile(pos1, pos2);
+
+    time = 0;
+    CPPUNIT_ASSERT_DOUBLES_EQUAL(pos1, v.Pos(time), epsilon);
+
+    time = 1.0;
+    CPPUNIT_ASSERT_DOUBLES_EQUAL(pos2, v.Pos(time), epsilon);
+
+    time = -1.0;
+    CPPUNIT_ASSERT_DOUBLES_EQUAL(pos2, v.Pos(time), epsilon);
```

Comment:
Shouldn't this be `pos1`?

Reply:
@smits Updated the source code and unit-test.